templates/front/my_checks_desktop.html

```diff
@@ -116,3 +125,5 @@
     {% endfor %}
 
 </table>
+
+
```

Comment:
why two blank lines? remove one of them.

Reply:
I have sorted this issue too.